src/components/Filters.jsx

```diff
@@ -0,0 +1,62 @@
+import "../styles/filters.css"
+import {useState} from "react"
+import { useFilter } from "../contexts/FilterContext"
+import { useAuth } from "../contexts/AuthContext"
+
+export default function Filters() {
+
+    const {filterState, filterDispatch} = useFilter()
+    const {loggedIn} = useAuth()
+    const [showFilters, setShowFilters] = useState(false)
+ 
+    return (
+        
+        <div className='filter-by'>
+            {loggedIn &&
+            <>
+            <p className="filter-title" onClick={() =>setShowFilters(!showFilters)}>Filter:</p>
```

Comment:
instead of doing this you can create your private route provider

Reply:
Sure, will explore the option and make the necessary changes.